### PR TITLE
database schema SQL file created

### PR DESCRIPTION
**Pull request by Scott-Guinn**
Created a schema.sql file which contains our schema for our postgres database.

Tested via psql \i command. 

Note: current default calorie count is 2000, can be removed if we don't want a default number in there (and do validation on input).

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+DROP DATABASE IF EXISTS nvision;
+CREATE DATABASE nvision;
+\c nvision;
+
+DROP TABLE IF EXISTS users CASCADE;
+CREATE TABLE users
+(
+ id          serial NOT NULL PRIMARY KEY,
```
Comment (taylorsmart) [inline schema.sql:8]: Huh - didn't know you could specify primary key like this.  This is cool


I'm so used to declaring it at the bottom of the schema 
PRIMARY KEY (ID)

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ firstName   varchar(50),
+ lastName    varchar(50),
+ username    varchar(50) NOT NULL,
+ password    varchar(50) NOT NULL,
+ calorieGoal smallint NOT NULL DEFAULT 2000,
+ waterGoal   smallint NOT NULL,
+ weightGoal  smallint,
+ phone       numeric,
+ email       varchar(50) NOT NULL,
+ sex         varchar(50)
```
Comment (taylorsmart) [inline schema.sql:18]: Thoughts on adding age to this?

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+CREATE TYPE mealType_t as enum('food', 'water');
+CREATE TYPE mealName_t as enum('breakfast', 'lunch', 'dinner', 'snack');
+
+DROP TABLE IF EXISTS entries CASCADE;
+
+CREATE TABLE entries
+(
+ id             serial PRIMARY KEY,
+ type           mealType_t,
```
Comment (taylorsmart) [inline schema.sql:29]: I haven't seen this type specification before - what does it do?

**Review by taylorsmart — APPROVED** [review]

**Review by emmaknor — APPROVED** [review]

File: schema.sql (hunk 2 above)
Comment (Scott-Guinn) [reply schema.sql:18]: what would we do with that data?

**Pull request merged**